data_model/src/expression.rs

```diff
@@ -338,6 +363,7 @@ impl ContextValue {
 }
 
 impl From<ContextValue> for ExpressionBox {
+    #[inline]
```

Comment:
`<Expression as Into>::into` can be arbitrarily complex, so this is not a great candidate.